pom.xml

```diff
@@ -26,10 +26,11 @@
         <artifactId>kafka-connect-parent</artifactId>
         <version>2.8.0-1</version>
     </parent>
+    <groupId>com.github.efcloud</groupId>
     <artifactId>kafka-config-provider-aws</artifactId>
-    <version>0.1-SNAPSHOT</version>
+    <version>0.1.0</version>
```

Comment:
It was likely set to SNAPSHOT on purpose, to avoid publishing it outside of the Releases cycle. The CI pipeline should set the right version before deploying it.